level_complete.gd

```diff
@@ -0,0 +1,11 @@
+extends Area2D
+export var last_level = ""
```

Comment:
change this to `export(PackedScene) var last_level`

Reply:
Actually, this should be a preload and be a victory scene which you'll have to create

Reply:
So steps:
- [ ] Create victory scene
- [ ] Save it
- [ ] Get resource path
- [ ] `var last_level = preload(NODE_PATH_GOES_HERE)`

Reply:
Oh okay